content/en/docs/concepts/containers/container-lifecycle-hooks.md

```diff
@@ -56,7 +56,8 @@ Resources consumed by the command are counted against the Container.
 ### Hook handler execution
 
 When a Container lifecycle management hook is called,
-the Kubernetes management system executes the handler in the Container registered for that hook. 
+the Kubernetes management system will execute handler according to the hook action,
+`exec` and `tcpSocket` will be executed in the container, and `httpGet` will be executed by the kubelet process.
```

Comment:
Use [present tense](https://kubernetes.io/docs/contribute/style/style-guide/#use-present-tense):

```suggestion
`exec` and `tcpSocket` are executed in the container, and `httpGet` is executed by the kubelet process.
```

Reply:
Modify according to [present tense](https://kubernetes.io/docs/contribute/style/style-guide/#use-present-tense). 
Thank you for your advice,please review again.